supabase/functions/_shared/helpers.ts

```diff
@@ -1,20 +1,51 @@
-import { corsHeaders } from "./cors.ts";
+import HandlebarsJS from 'https://esm.sh/handlebars';
+import Mustache from 'https://esm.sh/mustache';
+import { corsHeaders } from './cors.ts';
 
 export const returnPostgresError = (error: any) => {
-  return new Response(JSON.stringify({ error }), {
-    headers: {
-      ...corsHeaders,
-      "Content-Type": "application/json",
-    },
-    status: 400,
-  });
+    return new Response(JSON.stringify({ error }), {
+        headers: {
+            ...corsHeaders,
+            'Content-Type': 'application/json',
+        },
+        status: 400,
+    });
 };
 
 export const handlebarsHelpers = {
-  urlencode: function(s: string) {
-    return encodeURIComponent(s)
-  },
-  basicauth: function(user: string, password: string) {
-    return btoa(`${user}:${password}`);
-  }
-}
+    urlencode: function (s: string) {
+        return encodeURIComponent(s);
+    },
+    basicauth: function (user: string, password: string) {
+        return btoa(`${user}:${password}`);
+    },
+};
+
+export const compileTemplate = (template: string, data: any, connector_id: string) => {
```

Comment:
Move the compile call to a stand alone function to keep logging/compare consistent.